wiki/History_of_osu!/osu!wiki/en.md

```diff
@@ -1,31 +1,31 @@
-# osu!wiki history
+# osu!wiki
 
 ## MediaWiki (2011 - 2017)
```

Comment:
Not 2011 - 2016?

Reply:
There was a fading transition time where MediaWiki and GitHub co-existed at the same time. The one that should be fixed is the other heading.